api.rb

```diff
@@ -1,12 +1,14 @@
 require 'sinatra'
 require 'jwt'
+require 'http'
+require 'active_support/core_ext/hash/indifferent_access'
 require_relative 'lib/notify'
 
 class MyApi < Sinatra::Base
   before do
     halt 401 if request.env['HTTP_AUTHORIZATION'].nil?
     acces_token = request.env['HTTP_AUTHORIZATION'].slice(7..-1)
-    JWT.decode acces_token, public_key, true, { algorithm: 'RS256' }
+    JWT.decode acces_token, nil, true, { algorithm: 'RS256', jwks: ->(options) { jwks(options) } }
```

Comment:
We could fix spelling of `access_token` while we're here

Reply:
done.